src/test/resources/io/github/pixee/maven/operator/test/pom-hack23-cia-result.xml

```diff
@@ -1788,7 +1788,7 @@
                 <phase>generate-sources</phase>
                 <configuration>
                   <target>
-                    <mkdir dir="${project.build.directory}/dependency"></mkdir>
+                    <mkdir dir="${project.build.directory}/dependency"></mkdir>>
```

Comment:
I'd also check why this extra `>` happened.

Reply:
@aldrinleal done

Reply:
Awesome. btw I already approved it.

Reply:
@aldrinleal we're going to bring this java source code into the codemodder repository, I have a draft [here](https://github.com/carlosu7/codemodder-java/pull/1) I just need to update it with the last 2 commits from here and also change the destination. When I do that, I will open a PR for it. (cc @nahsra )